content/sensu-go/5.15/api/federation.md

```diff
@@ -0,0 +1,442 @@
+response codes            | <ul><li>**Success**: 204 (No Content)</li><li>**Missing**: 404 (Not Found)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+
+
+## The `/clusters` endpoint
+
+### `/clusters` (GET)
+
+The `/clusters` endpoint provides HTTP GET access to a list of clusters.
+
+#### EXAMPLE {#clusters-get-example}
+
+The following example demonstrates a request to the `/clusters` endpoint, resulting in a list of clusters.
+
+{{< highlight shell >}}
+curl -H "Authorization: Bearer $SENSU_TOKEN" \
+http://127.0.0.1:8080/api/enterprise/federation/v1/clusters
+
+HTTP/1.1 200 OK
+
+[
+    {
+        "type": "Cluster",
+        "api_version": "federation/v1",
+        "metadata": {
+            "name": "us-west-2a"
+        },
+        "spec": {
+            "api_urls": [
+                "http://10.0.0.1:8080",
+                "http://10.0.0.2:8080",
+                "http://10.0.0.3:8080"
+            ]
+        }
+    }
+]
+{{< /highlight >}}
+
+#### API Specification {#clusters-get-specification}
+
+/clusters (GET)  | 
+---------------|------
+description    | Returns a list of clusters.
+example url    | http://hostname:8080/api/enterprise/federation/v1/clusters
+response type  | Array
+response codes | <ul><li>**Success**: 200 (OK)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+output         | {{< highlight shell >}}
+[
+    {
+        "type": "Cluster",
+        "api_version": "federation/v1",
+        "metadata": {
+            "name": "us-west-2a"
+        },
+        "spec": {
+            "api_urls": [
+                "http://10.0.0.1:8080",
+                "http://10.0.0.2:8080",
+                "http://10.0.0.3:8080"
+            ]
+        }
+    }
+]
+{{< /highlight >}}
+
+## The `/clusters/:cluster` endpoint {#the-clusterscluster-endpoint}
+
+### `/clusters/:cluster` (GET) {#clusterscluster-get}
+
+The `/clusters/:cluster` endpoint provides HTTP GET access to data for a specific `cluster`, by cluster `name`.
+
+#### EXAMPLE {#clusterscluster-get-example}
+
+In the following example, querying the `/clusters/:cluster` endpoing returns a JSON Map containing the requested `:etcd-replicator`.
+
+{{< highlight shell >}}
+curl -H "Authorization: Bearer $SENSU_TOKEN" \
+http://127.0.0.1:8080/api/enterprise/federation/v1/clusters/us-west-2a
+
+HTTP/1.1 200 OK
+
+{
+  "type": "Cluster",
+  "api_version": "federation/v1",
+  "metadata": {
+      "name": "us-west-2a"
+  },
+  "spec": {
+      "api_urls": [
+          "http://10.0.0.1:8080",
+          "http://10.0.0.2:8080",
+          "http://10.0.0.3:8080"
+      ]
+  }
+}
+{{< /highlight >}}
+
+#### API Specification {#etcd-replicatorsetcd-replicator-get-specification}
+
+/clusters/:cluster (GET) | 
+---------------------|------
+description          | Returns the specified cluster.
+example url          | http://hostname:8080/api/enterprise/federation/v1/clusters/{cluster_name}
```

Comment:
In the other endpoints, you used the literal `us-west-2a` name. I would stay consistent here.
```suggestion
example url          | http://hostname:8080/api/enterprise/federation/v1/clusters/us-west-2a
```